libs/novui/src/recipes/loading-overlay.recipe.ts

```diff
@@ -0,0 +1,33 @@
+import { LoadingOverlayStylesNames } from '@mantine/core';
+import { defineSlotRecipe } from '@pandacss/dev';
+import { colorPaletteGradientHorizontal } from 'src/ingredients';
```

Comment:
this will not compile I believe :) should be `../ingredients`

Reply:
Thank you! I hate that my editor always does this, and haven't found a solution that consistently addresses this